### DISCUSSION TITLE
same email but different provider

### DISCUSSION THREAD

**Discussion by artifacts3** [supabase/supabase#11381] [Questions]
example 
- google email: rhrlgns777@gmail.com
- apple email: rhrlgns777@gmail.com

what should i do provider two login?

**Comment by GaryAustin1** [comment]
What are you trying to do?
Right now either of those logins should access the same user based on the common email.  Supabase uses either phone or email as the way to link users.

**Comment by artifacts3** [comment]
i have google, apple account ***@***.***

and i was login google provider in my service supabase by
***@***.***

but i want to login apple provider in my service supabase by
***@***.***


2023년 1월 1일 (일) 오전 1:43, GaryAustin1 ***@***.***>님이 작성:

**Comment by artifacts3** [comment]
how do i can base on email and provider type?? not only email

2023년 1월 1일 (일) 오후 2:56, Kihoon Ko ***@***.***>님이 작성:
